kolibri/logger/models.py

```diff
@@ -98,10 +101,16 @@ class ContentSessionLog(BaseLogModel):
     start_timestamp = DateTimeTzField()
     end_timestamp = DateTimeTzField(blank=True, null=True)
     time_spent = models.FloatField(help_text="(in seconds)", default=0.0, validators=[MinValueValidator(0)])
-    progress = models.FloatField(default=0, validators=[MinValueValidator(0)])
+    progress = models.FloatField(default=0, validators=[MinValueValidator(0), MaxValueValidator(1.0)])
```

Comment:
My understanding of when this would happen was e.g. if someone watched a video twice through within a session. We didn't want to reflect that on the summary log, but on the session log we could reflect that fact. Not essential to keep, but it's possible that logic has been maintained somewhere, so adding this validator could cause saving to blow up and logs not to save?

(The data migration will only apply against old data)

Reply:
Yup, should be 1.01 as well? I'm wondering what level of "blowing up" is desirable :)

Reply:
Oh, I see your logic. But I believe progress on a session log could be arbitrarily high (due to what I describe in the parent comment), not just 1.01.